openmetadata-service/src/main/resources/elasticsearch/en/database_schema_index_mapping.json

```diff
@@ -306,7 +306,7 @@
             }
           },
           "fullyQualifiedName": {
-            "type": "text"
+            "type": "keyword"
```

Comment:
@mgorsk1 Thanks for the contribution. Can you follow the same pattern for fullyQualifiedName similar to other indexes 
```"fullyQualifiedName": {
        "type": "keyword",
        "normalizer": "lowercase_normalizer"
      },```
and also this change needed for other languages as well